include/krml/internal/target.h

```diff
@@ -57,6 +57,18 @@
 #  define KRML_HOST_IGNORE(x) (void)(x)
 #endif
 
+#ifndef KRML_NOINLINE
+#  if defined(_MSC_VER)
+#    define KRML_NOINLINE __declspec(noinline)
+#  elif defined (__GNUC__)
```

Comment:
clang also sets `__GNUC__`, but we could use `|| __llvm__` to be sure to cover both.